packages/mongo/mongo_driver.js

```diff
@@ -506,6 +506,8 @@ MongoConnection.prototype._update = function (collection_name, selector, mod,
   try {
     var collection = self.rawCollection(collection_name);
     var mongoOpts = {safe: true};
+    // Add support for filtered positional operator
+    if (_.has(options, 'arrayFilters')) mongoOpts.arrayFilters = options.arrayFilters;
```

Comment:
Is Underscore really needed here? Wouldn't
```js
options.hasOwnProperty('arrayFilters')
```
be enough?

Reply:
You're right. I saw Underscore being used this way in a number of files, so I assumed it was some code convention. I've replaced it with your suggestion

Reply:
As it turns out, using `hasOwnProperty` instead of Underscore causes test failures? I'm not sure how to proceed in this situation, should I just change it back?

Reply:
might be because `options` might be `null` and underscore is able to handle that?

Reply:
That could be it, but the next couple of lines, using `options.upsert` and `options.multi`

Reply:
🤔 you're right... awesome PR btw, can't wait to use `arrayFilters` without  relying on `rawCollection()`, it's a bit a bit awkward to bypass Meteor collection when using `redis-oplog`